lib/xml.js

```diff
@@ -198,7 +175,7 @@ function convert (zObjectType, objects) {
  * I thought the ZOQL docs said that keywords needed to be lowercase.
```

Comment:
This is actually what the "Zuora Object Query Language"  documentation says : https://knowledgecenter.zuora.com/BC_Developers/SOAP_API/M_Zuora_Object_Query_Language